tests/test_find_batched.py

```diff
@@ -0,0 +1,99 @@
+# built-in dependencies
+import os
+
+# 3rd party dependencies
+import cv2
+import pandas as pd
+
+# project dependencies
+from deepface import DeepFace
+from deepface.modules import verification
+from deepface.commons import image_utils
+from deepface.commons.logger import Logger
+
+logger = Logger()
+
+
+threshold = verification.find_threshold(model_name="VGG-Face", distance_metric="cosine")
+
+
+def test_find_with_exact_path():
+    img_path = os.path.join("dataset", "img1.jpg")
+    results = DeepFace.find(img_path=img_path, db_path="dataset", silent=True, batched=True)
+    assert len(results) > 0
+    for result in results:
+        assert isinstance(result, list)
+
+        found_image_itself = False
+        for face in result:
+            assert isinstance(face, dict)
```

Comment:
you may consider to check required keys in face

Reply:
https://github.com/serengil/deepface/pull/1351/commits/3bb317ac63f87624bd8b77d76e919d42edff13c5#diff-c9703fe704ffb9ab295563a197a264d2d938a16b4dd67bacd9cdb5ee26a85b8dR22-R26